veggieseasons/lib/screens/details.dart

```diff
@@ -70,14 +70,14 @@ class ServingInfoChart extends StatelessWidget {
                       TableCell(
                         child: Text(
                           'Serving size:',
-                          style: Styles.detailsServingLabelText,
+                          style: Styles.detailsServingLabelText(context),
```

Comment:
There is a small cost to repeatedly using BuildContexts to look up an inherited widget, which is what the methods in styles.dart are doing with the context being passed here. Consider having those methods take a CupertinoThemeData directly, so you can fetch that data once in this class, and use it repeatedly in calls to the styles.

This would also help in a small way with separation of concern, since the styling really just needs the theme data itself, rather than full access to the build context in most cases.